docs/reference/modules/cluster/disk_allocator.asciidoc

```diff
@@ -113,6 +113,19 @@ PUT /my-index-000001/_settings
 --
 // end::cluster-routing-flood-stage-tag[]
 
+[[cluster-routing-flood-stage-frozen]]
+// tag::cluster-routing-flood-stage-tag[]
+`cluster.routing.allocation.disk.watermark.flood_stage.frozen` {ess-icon}::
+(<<dynamic-cluster-setting,Dynamic>>)
+Controls the flood stage watermark for frozen nodes, which defaults to 95%.
```

Comment:
for "dedicated frozen nodes"?

Reply:
Thanks, fixed in 22e49ed90dcbe78f18825328af8d7521ef0c06b3